.coveralls.yml

```diff
@@ -0,0 +1,2 @@
+service_name: travis-pro
+repo_token: ugkXbvXKZado9bNk1IVMdHhe6IWBrgNjI
```

Comment:
Do not hardcode the repo_token value. Use environment variable on travis instead

Reply:
Ok, let me do it